sw/airborne/firmwares/rotorcraft/navigation.c

```diff
@@ -482,16 +486,20 @@ void nav_home(void)
 
 /** Set manual roll, pitch and yaw without stabilization
  *
- * @param[in] roll The angle in radians (float)
- * @param[in] pitch The angle in radians (float)
- * @param[in] yaw The angle in radians (float)
+ * @param[in] roll command in pprz scale (float)
+ * @param[in] pitch command in pprz scale (float)
+ * @param[in] yaw command in pprz scale (float)
+ *
+ * This function allows to directly set commands from the flight plan,
+ * if in nav_manual mode.
+ * This is for instance useful for helicopters during the spinup
  */
 void nav_set_manual(float roll, float pitch, float yaw)
```

Comment:
why have roll, pitch, yaw as floats here when the commands are always in int32_t anyway?